### DISCUSSION TITLE
Bug: Uploading multiple files to a release in parallel causes nonsense

### DISCUSSION THREAD

**Discussion by LinqLover** [community/community#13911] [Other Feature Feedback, Questions, & Ideas]
While the upload of my first build file to the release I was drafting was still in progress, I dropped a second file onto the progress bar. As a consequence, the progress bar started bouncing between two positions, and eventually, the second file was not uploaded, but it was not possible for me to save the draft for the release. After reloading the page both files were displayed, but I needed to delete the second file and add it again in order to remove the error icon from it.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by LinqLover** [reply]
Thanks, done
